docs/sql-ref-ansi-compliance.md

```diff
@@ -111,6 +111,8 @@ SELECT * FROM t;
 
 The behavior of some SQL functions can be different under ANSI mode (`spark.sql.ansi.enabled=true`).
   - `size`: This function returns null for null input under ANSI mode.
+  - `element_at`: This function throws `ArrayIndexOutOfBoundsException` if using invalid indices under ANSI mode.
+  - `elt`: This function throws `ArrayIndexOutOfBoundsException` if using invalid indices under ANSI mode.
```

Comment:
I think its better to describe the behaviour change of `GetArrayItem`, too, so how about creating a new subsection for it like `Other SQL Operations`?

Reply:
nit: how about removing "`under ANSI mode`" in each entry? They look redundant.

Reply:
+1